zipline/pipeline/factors/technical.py

```diff
@@ -598,7 +598,7 @@ class IchimokuKinkoHyo(CustomFactor):
         'kijun_sen_length': 26,
         'chikou_span_length': 26,
     }
-    inputs = USEquityPricing.high, USEquityPricing.close
+    inputs = [USEquityPricing.high, USEquityPricing.low, USEquityPricing.close]
```

Comment:
I fixed "1" but it's not straightforward to add a test for "2", no other factor has that test.